internal/app/machined/pkg/controllers/network/status.go

```diff
@@ -142,7 +142,7 @@ func (ctrl *StatusController) Run(ctx context.Context, r controller.Runtime, log
 		// etc files
 		result.EtcFilesReady = true
 
-		for _, requiredFile := range []string{"hosts", "resolv.conf"} {
+		for _, requiredFile := range []string{"hosts", "resolv.conf", "resolv-global.conf"} {
```

Comment:
I think it's less relevant for the "network up" status.

Reply:
Removed.